src/main/java/de/tum/in/www1/artemis/domain/FeedbackConflict.java

```diff
@@ -34,6 +34,9 @@
     @JoinColumn(name = "second_feedback_id", referencedColumnName = "id")
     private Feedback secondFeedback;
 
+    @Column(name = "markedAsNotConflict")
+    private Boolean markedAsNotConflict;
```

Comment:
I though this is already achieved by the variable `conflict` above? why do we need another boolean flag?

Reply:
This boolean field (`markedAsNotConflict`) basically means, there is a conflict according to Athene but tutor thinks it is not a conflict.

Reply:
Hmm I don’t really like the name. Also you don’t follow the naming conventions with _

Reply:
I changed it to `discard`